sdk/storage/azure-storage-file-datalake/setup.py

```diff
@@ -93,7 +93,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.0.0",
         "msrest>=0.6.10",
-        "azure-storage-blob~=12.0.0"
```

Comment:
Can we instead place a dev dependency on `azure-storage-blob_12.0.1b6` or the like? I have a feeling it would still match.